src/main/java/org/schemaspy/LayoutFiles.java

```diff
@@ -0,0 +1,48 @@
+package org.schemaspy;
+
+import org.apache.commons.io.filefilter.FileFilterUtils;
+import org.apache.commons.io.filefilter.IOFileFilter;
+import org.schemaspy.util.ResourceWriter;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.Objects;
+
+/**
+ * Represents the contents of the layout folder.
```

Comment:
If it represents the content of the layout folder, why should it know about the output dir?

Or are we suggesting outputDir is the layout folder?

Reply:
Let me rephrase, at some point it could know about the outputDir. But having it part of the constructor makes this, not a representation of Layout Content, but a CopyFilesTask.

We create this object twice, they differ in the outputdir, not in the layout content. But that's probably a discussions for the later PR's.